app/pages/project/project-navbar.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { Component, PropTypes } from 'react';
+import Translate from 'react-translate-component';
+import { IndexLink, Link } from 'react-router';
+import classnames from 'classnames';
+import partition from 'lodash/partition';
+import SOCIAL_ICONS from '../../lib/social-icons';
+import Thumbnail from '../../components/thumbnail';
+
+// constants
+const AVATAR_SIZE = 100;
+
+export default class ProjectNavbar extends Component {
+  constructor() {
```

Comment:
You don't need to bind any of the component methods, so you can delete the constructor. 

In fact, since the component has no state, it can be written as a function rather than a class.

Reply:
That's how I started actually, but kept getting linting/syntax errors. Who knows, I'll try again.

Reply:
TBH, don't waste your time looking at this, until I get rid of the WIP label.

Reply:
👍🏻